docs/notebooks/pulse-efficient-transpilation.ipynb

```diff
@@ -498,17 +553,17 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 21,
+   "execution_count": 15,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
       "ECR transpile: two-qubit circuit depth = 40\n",
-      "ECR circuit duration: 30.06 us\n",
+      "ECR circuit duration: 21.084 us\n",
       "RZX transpile: two-qubit circuit depth = 20\n",
```

Comment:
I reran the notebook with qiskit version = 1.1.0, and I got a two-qubit circuit depth = 40 for the RZX-based transpilation - however, it should be `20` instead as the RZX-based transpilation should produce a two-qubit depth hold of the ECR-based transpilation. Could you check if that is the case for you as well?

Reply:
I reran the notebook with 1.0.0 and optimization level = 1 and got this: 

ECR transpile: two-qubit circuit depth = 40
ECR circuit duration: 30.06 us
RZX transpile: two-qubit circuit depth = 20
RZX circuit duration: 6.220000000000001 us